stages/test/test_mkdir.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/python3
+
+import contextlib
+import os
+import subprocess
+
+import pytest  # type: ignore
+
+from osbuild.testutil import has_executable
+
+STAGE_NAME = "org.osbuild.mkdir"
+
+
+def test_mkdir(tmp_path, stage_module):
+    tree = tmp_path / "tree"
+    tree.mkdir()
+
+    options = {
+        "paths": [
+            {"path": "/fake_dir"},
+            {"path": "fake_relative_dir"}
+        ]
+    }
+    args = {
+        "tree": f"{tree}",
```

Comment:
(nitpick) `"tree": tree.as_posix()` would also be an option but either is fine

Reply:
I'm just used to `f"{..}"` syntax

Reply:
Its totally fine, we are not super consistent here, you can keep it if you prefer it :)